goodtablesio/models/job.py

```diff
@@ -110,15 +113,53 @@ def get_ids():
     return [j.id for j in job_ids]
 
 
-def get_all():
-    """Get all jobs in the database as dict.
+def find(filters=None, limit=10, offset=0):
```

Comment:
I have nothing against the concept of helpers in general of course. So my idea is to use it when it reduce complexity and don't use when it increase complexity (as you said we will see).